frontend/src/Search/SearchIndex.js

```diff
@@ -192,7 +196,9 @@ class SearchIndex extends Component {
 
   onKeyUp = (event) => {
     const jumpBarItems = this.state.jumpBarItems.order;
-    if (event.path.length === 4) {
+    const path = event.composedPath ? event.composedPath() : event.path;
+
+    if (path.length === 4) {
```

Comment:
Can we just use below, it's what we've standardized on in other places, seems like most all modern browsers support

```
if (event.composedPath && event.composedPath().length === 4) {
```